src/interface/others/AverageTargetsHit.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class AverageTargetsHit extends React.PureComponent {
+  static propTypes = {
+    casts: PropTypes.string.isRequired,
+    hits: PropTypes.string.isRequired,
+    approximate: PropTypes.bool,
```

Comment:
Thoughts on removing this prop and instead using the math in render() to determine if you're showing a rounded number or an accurate number?

Reply:
The prop is used in all similiar versions of this (ItemDamageDone etc)

Reply:
That's true, but I also can't find any instances of the prop actually being passed for any of those. So maybe it's worth also removing it from those and applying the same logic to them?

Reply:
If you still wanna keep the prop (I personally don't mind), you might wanna set `defaultProps` to make it clear if it's by default true or false - adding this to the end (outside class)
```javascript
AverageTargetsHit.defaultProps = {
  approximate: true,
};
```

Reply:
I should have made it clear too that my comments weren't a blocking review either, that's why I didn't set to Requested Changes. I was hoping for some discussion about the whole idea of that prop -- as we can see that even in the existing modules it is never used. In this very specific case I feel like because the math is actually being done inside the module itself it probably is an opportune time to not include the prop and use some logic to determine whether or not to include the approximation symbol. 

What would approximate even mean when passed to this module? Does someone using it have to check whether or not hits and casts are factors and then decide whether or not to set approximate to true? If they don't it could show ~1.0 casts, even if it's an exact number. I feel like the ~ should only appear if the number shown was actually rounded after dividing.